.github/workflows/performance-tests.yml

```
@@ -0,0 +1,584 @@
+            TESTS="ticket-sales,check-in,sustained"
+          elif [ "${{ github.event_name }}" = "push" ] && [ "${{ github.ref }}" = "refs/heads/main" ]; then
+            TESTS="ticket-sales,check-in"
+          else
+            # Pull request - run lightweight tests
+            TESTS="ticket-sales"
+          fi
+          echo "tests=${TESTS:-ticket-sales,check-in}" >> $GITHUB_OUTPUT
+          
+      - name: Determine target URL
+        id: determine-url
+        run: |
```

actionlint (1.7.7)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)